tests/conftest.py

```diff
@@ -0,0 +1,51 @@
+import base64
+import json
+import os
+
+import pytest
+import requests
+
+
+@pytest.fixture(scope="session")
```

Comment:
This is pretty cool! Just read up on test fixtures and seems like exactly what should be used here.